### ISSUE TITLE
How to use BarcodeResult.Raw

### ISSUE THREAD

**Issue by kidmar** [Redth/ZXing.Net.Maui#36]
I need to know if the barcode contains some special characters, like GS for a GS1-128 barcode.
I tried decoding the Raw property, but I cannot find a way to understand what's inside that byte[].
I would need to translate it to ascii characters to check for GS (ASCII 29). Is it possible?

**Comment by g0dpain** [comment]
I don't know what GS means, but scanning [this barcode](https://barcode.tec-it.com/de/EANUCC128) and access the property ``Value`` I get "0101234567890128TEC-IT"
Translating the byte[] into a string in either encoding it is not readable

**Comment by kidmar** [comment]
Yes that's the problem. GS is a not readable character, like Enter but it has a specific meaning in barcodes (separate variable length informations).
The byte[] should contain the ASCII code of every character in the barcode, but we need to know how to read it

**Comment by g0dpain** [comment]
The output of the barcode mentioned before is
 if-CYd4%#
)4+j

You can test it [here](https://zxing.org/w/decode.jspx)
This is also the same output when using the [original source project](https://github.com/zxing/zxing)

If no one here has an answer you definitively should ask there

**Comment by jfversluis** [comment]
More a how-to question than a potential issue with the SDK. Closing this here for now.
